scripts.js

```diff
@@ -0,0 +1,98 @@
+const myLibrary = [];
+let count = 0;
+const container = document.querySelector('#books');
+
+function appendElement(element, parent, classElement = '') {
+  const child = document.createElement(element);
+  parent.appendChild(child).className = classElement;
+  return child;
+}
+
+function readBook(readButton) {
+  readButton.addEventListener('click', () => {
+    const parent = readButton.parentElement;
+    const { children } = parent;
+    if (myLibrary[readButton.value - 1].read === 'Read') {
+      readButton.textContent = 'Mark as read';
+      myLibrary[readButton.value - 1].read = 'Unread';
+      children[3].textContent = 'Unread';
+    } else {
+      readButton.textContent = 'Mark as unread';
+      myLibrary[readButton.value - 1].read = 'Read';
+      children[3].textContent = 'Read';
+    }
+  });
+}
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/